dotCMS/src/main/webapp/dotcms-webcomponents/p-7he2yafe.system.entry.js

```diff
@@ -1 +1 @@
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid assignments in operands](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5231770)